chart/templates/otel-auto-instrumentation.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.opentelemetryOperator.enabled -}}
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: tobs-auto-instrumentation
+spec:
+  exporter:
+    # TODO(paulfantom): check if endpoint is correct
+    endpoint: http://{{ .Release.Name }}-otel-collector.{{ .Release.Namespace }}.svc:4317
```

Comment:
Shouldn't we include `.cluster.local`?

Reply:
No, `.cluster.local` is customizable by kubernetes cluster admin. For manifests that need to be portable, it shouldn't be specified.